distribution/docs/src/main/resources/content/_using/workspaces.adoc

```diff
@@ -0,0 +1,67 @@
+:title: Using Workspaces
+:type: using
+:status: published
+:parent: Using ${catalog-ui}
+:summary: Using workspaces within ${catalog-ui}
+:order: 03
+
+== {title}
+
+Within ${catalog-ui}, workspaces are collections of settings, searches, and bookmarks that can be shared between users and stored for repeated access.
+
+=== Creating a Workspace in ${catalog-ui}
+
+Before searching in ${branding}, at least one workspace must be created.
+
+.Start new workspace
+. From the *Workspaces* view, enter search terms into the *Start new workspace* search field and click the magnifying glass (image:magnifying-glass.png[magnifying glass icon]) icon. This will create a new workspace and perform a search based on the entered search terms.
+
+.Start new workspace
+image::start-new-workspace.png[start new workspace icon]
+
+=== Configuring a Workspace in ${catalog-ui}
+
+Configure each workspace with searches and share options.
+
+.Adding searches
+. From the default *Workspaces* view, select the workspace to add a search to.
+. Click *Search ${branding} Intrigue* in the upper left corner, enter search terms, and click *Search* to add a search. This step can be repeated to add additional searches. Each workspace can have up to ten searches.
+.. Select *Basic* Search to select simple search criteria, such as *text*, *time*, and *location*.
```

Comment:
Might be worth noting you select other query forms from the ellipsis menu

Reply:
I can address these on this current PR, but this issue, #5035, is going to be verifying and revising this content